microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -153,7 +153,15 @@ To rename your new Snippet with a custom name, navigate to [Rename Snippets](#re
 
 1.  [Open the Snippets tab](#open-the-snippets-tab).  
 1.  Hover on the Snippet name, open the contextual menu \(right-click\), and choose **Remove**.  
-    
+
+## Save Snippets  
+
+By default, Snippets are only available within DevTools, but you can also save them to disk.  
+
+1.  [Open the Snippets tab](#open-the-snippets-tab).  
+1.  Hover on the Snippet name, open the contextual menu \(right-click\), and choose **Save as...**.  
```

Comment:
```suggestion
1.  Hover on the Snippet name, open the contextual menu \(right-click\), and select **Save as**.  
```
You might want to Edit this Suggestion to continue to use 'choose' per rest of article.  But, "..." should be deleted, per Microsoft Writing Style Guide.  Public version of "Ellipses" page https://docs.microsoft.com/en-us/style-guide/punctuation/ellipses says "When there's an ellipsis in UI, don’t include it in instructions or procedures.  Example: Select Safety, and then Delete browsing history."